### PR TITLE
fix: Revert Faker removal, add Faker seed

### PR DESCRIPTION
**Pull request by seankeating**
This reverts the Faker.JS removal and puts a faker.seed() in the index.tsx to keep faker calls consistent between page calls.

### CODE REVIEW

File: test/specs/commonTests/isConformant.tsx [hunk 1]
```diff
@@ -411,7 +412,7 @@ export default (Component, options: any = {}) => {
 
       if (!defaultClasses) return
 
-      const userClasses = 'generate'
+      const userClasses = faker.hacker.verb()
```
Comment (levithomason) [inline test/specs/commonTests/isConformant.tsx]: Faker in tests can cause intermittent or hard to reproduce failures.  Let's only use it in the docs.

Comment (levithomason) [inline test/specs/commonTests/isConformant.tsx:415]: Faker in tests can cause intermittent or hard to reproduce failures.  Let's only use it in the docs.

File: test/specs/commonTests/rendersChildren.tsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import faker from 'faker'
```
Comment (levithomason) [inline test/specs/commonTests/rendersChildren.tsx:1]: As above, let's only use faker in docs.

File: test/specs/lib/AutoControlledComponent-test.tsx [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import faker from 'faker'
```
Comment (levithomason) [inline test/specs/lib/AutoControlledComponent-test.tsx:1]: In docs only.

**Review by levithomason — APPROVED** [review]
Remove from test and merge.

**Comment by levithomason** [comment]
Bump, we need to remove faker from the tests here before merge.

**Comment by seankeating** [comment]
This solution has issues, and the faker.seed(...) is not consistent.  You can see on first load of a doc page of a component, and when showing the code for a component.

**Comment by seankeating** [comment]
This solution has issues, and the faker.seed(...) is not consistent.  You can see on first load of a doc page of a component, and when showing the code for a component.

**Pull request closed without merging**